### PR TITLE
Add DTVA custom kernel

### PR DESCRIPTION
**Pull request by briannwu**
Custom_Cijk_Alik_Bljk_F8BS_BH_BiasSB_AS_SABV_SAV_UserArgs_MT256x256x128_MI16x16x1_SN_K1_MIWT4_16_DTVA Custom_Cijk_Alik_Bljk_F8BS_BH_BiasSB_AS_SABV_SAV_UserArgs_MT256x256x128_MI16x16x1_SN_K1_MIWT4_16_WSGRB2_DTVA
Custom_Cijk_Alik_Bljk_F8BS_BH_BiasSB_AS_SABV_SAV_UserArgs_MT256x224x128_MI16x16x1_SN_K1_MIWT4_14_DTVA Custom_Cijk_Alik_Bljk_F8BS_BH_BiasSB_AS_SABV_SAV_UserArgs_MT256x224x128_MI16x16x1_SN_K1_MIWT4_14_WSGRB2_DTVA

### CODE REVIEW

**Comment by StaceyLai** [comment]
If your commit involves changes to a YAML file, please take note.

Due to PR-1117 (https://github.com/ROCm/hipBLASLt/pull/1117) adding `ActivationType: hipblaslt_all`, all YAML files now need to include the ActivationType field (for hipblaslt logic YAML, use `ActivationType: hipblaslt_all` (not `all`); for tensile test YAML, you can use either `all` or `hipblaslt_all`). 

Correspondingly, in the Logic YAML filename, the `_A` that originally represented `ActivationType: all` should also be changed to `_HA`. These changes have been implemented in PR-1117.

**Comment by alex391a** [comment]
In #1058 , I modified the kernel argument definition. These two files were generated before this change, so the sgprWGM value may be incorrect. 
Seems that adding 
`s_sext_i32_i16 s[sgprWGM], s[sgprWGM]              // Restore WGM`
after
`label_GSU_End:`
can resolve this issue.
(no need to add the wgmxcc calculation part)

tensilelite/Tensile/CustomKernels/Custom_Cijk_Alik_Bljk_F8BS_BH_BiasSB_AS_SABV_SAV_UserArgs_MT256x256x128_MI16x16x1_SN_K1_MIWT4_16_DTVA.s
tensilelite/Tensile/CustomKernels/Custom_Cijk_Alik_Bljk_F8BS_BH_BiasSB_AS_SABV_SAV_UserArgs_MT256x256x128_MI16x16x1_SN_K1_MIWT4_16_WSGRB2_DTVA.s

**Review by aazz44ss — DISMISSED** [review]
LGTM

**Comment by briannwu** [comment]
> In #1058 , I modified the kernel argument definition. These two files were generated before this change, so the sgprWGM value may be incorrect. Seems that adding `s_sext_i32_i16 s[sgprWGM], s[sgprWGM] // Restore WGM` after `label_GSU_End:` can resolve this issue. (no need to add the wgmxcc calculation part)
> 
> tensilelite/Tensile/CustomKernels/Custom_Cijk_Alik_Bljk_F8BS_BH_BiasSB_AS_SABV_SAV_UserArgs_MT256x256x128_MI16x16x1_SN_K1_MIWT4_16_DTVA.s tensilelite/Tensile/CustomKernels/Custom_Cijk_Alik_Bljk_F8BS_BH_BiasSB_AS_SABV_SAV_UserArgs_MT256x256x128_MI16x16x1_SN_K1_MIWT4_16_WSGRB2_DTVA.s

@alex391a done, thanks~

File: tensilelite/Tensile/Tests/common/gemm/gemm_ck_gfx942.yaml
Comment (hcman2) [inline tensilelite/Tensile/Tests/common/gemm/gemm_ck_gfx942.yaml:1]: file mode changed?

Comment (briannwu) [reply tensilelite/Tensile/Tests/common/gemm/gemm_ck_gfx942.yaml:1]: Fixed.

**Review by hcman2 — APPROVED** [review]

**Comment by briannwu** [comment]
Tensilelite yaml test pass in local: tensilelite/Tensile/Tests/common/gemm/gemm_ck_gfx942.yaml

**Review by Jinp800125 — APPROVED** [review]

**Pull request merged**